### PR TITLE
Feature/ritik/regenerate order

### PR DESCRIPTION
**Pull request by ritikatsp**

### CODE REVIEW

File: common/models/report-model.js [hunk 1]
```diff
@@ -1615,6 +1615,34 @@ module.exports = function (ReportModel) {
...
+                    functionName: 'regenerateOrder'
+                });
+                return Promise.reject('Could not find report model instance');
+            })
+            .then( function(response) {
+                reportModelInstance = response;
+                return reportModelInstance.updateAttributes({
+                    state: REPORT_STATES.GENERATED
+                });
+            })
```
Comment (kamal0808) [inline common/models/report-model.js:1643]: might need a catch block at the end

File: log.log
Comment (kamal0808) [inline log.log:1]: why do you need this file committed?

**Review by kamal0808 — CHANGES_REQUESTED** [review]

File: common/models/report-model.js (hunk 1 above)
Comment (ritikatsp) [reply common/models/report-model.js:1643]: it's there at 1628, followed the same code convention as used earlier in the file.

File: log.log
Comment (ritikatsp) [reply log.log:1]: removing

**Review by kamal0808 — APPROVED** [review]

**Pull request merged**